src/components/admin.ts

```diff
@@ -9,7 +9,7 @@ const MOD_USER_ID_FOR_BAN_APPEAL: string = vars.MOD_USER_ID_FOR_BAN_APPEAL;
 const makeBanMessage = (reason: string, days?: number): string =>
   `
 Uh oh, you have been banned from the UW Computer Science Club server ${
-    days && `and your messages in the past ${days} ${pluralize('day', days)} have been deleted`
+    days && `and your messages in the past ${days} ${pluralize('day', days)} have been deleted `
```

Comment:
Edit: It looks like this won't work as intended if `days` is undefined, so just revert it back to the ternary operator